core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java

```diff
@@ -256,7 +255,10 @@ private static ImageIcon loadIconForObjectMarker(Color color) {
   }
 
   private static Icon loadIconForCameraMarker(Color color) {
-    URL markerIconURL = StorytellingSceneEditor.class.getResource("images/markerIconGrayScale.png");
+    // TOD convert VR flag to parameter
```

Comment:
TODO?

Reply:
are you just checking if i'm reading the comments? 😂

Reply:
Would you believe Tod is the next developer to join our team?
I have this addressed in another branch so I will let my typo will stand until then.

Reply:
👍